src/components/withLocalize.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import getComponentDisplayName from '../libs/getComponentDisplayName';
+import compose from '../libs/compose';
+import ONYXKEYS from '../ONYXKEYS';
+import {translate} from '../libs/translate';
+import DateUtils from '../libs/DateUtils';
+import {toLocalPhone, fromLocalPhone} from '../libs/LocalePhoneNumber';
+import numberFormat from '../libs/numberFormat';
+
+const withLocalizePropTypes = {
```

Comment:
Any reason this is here instead of directly on the export?

Reply:
It's a common pattern throughout the codebase. See here https://github.com/Expensify/Expensify.cash/blob/e9e726aa2a2f24ae98e88d9aa166acf4e8036ecb/src/components/withWindowDimensions.js

Reply:
Oh ok, will ask around why we chose to do that. No need to change it then.